lib/foreman_theme_satellite/documentation.rb

```diff
@@ -77,28 +77,44 @@ module Documentation
       'foreman_discovery' => "#{ForemanThemeSatellite.documentation_root}/provisioning_hosts/discovering-hosts-on-a-network_provisioning",
     }.freeze
 
-    DOCS_GUIDES_LINKS = {
-      'Managing_Content' => {
-        'Products_and_Repositories_content-management' => "#{ForemanThemeSatellite.documentation_root}/managing_content/importing_content_content-management#Products_and_Repositories_content-management",
-      },
+    # Guide mapping. If no entry is present, it should be assumed that it can
+    # be downcased
+    DOCS_GUIDE_MAPPING = {
+      'Managing_Configurations_Ansible' => 'managing_configurations_using_ansible_integration',
+    }.freeze
+
+    # An upstream chapter mapping, in case downstream another chapter should be
+    # used. The top level key is the upstream guide name where the value is a
+    # mapping of upstream chapter to downstream mapping
+    DOCS_GUIDE_CHAPTER_MAPPING = {
```

Comment:
I think it's questionable if we really want this, but I kept it in since we had this.